eng/scripts/tests/Update-SdkVersion.constants.ps1

```diff
@@ -0,0 +1,81 @@
+$AzVersionLocation = "$PSScriptRoot../../../../sdk/inc/azure/core/az_version.h"
```

Comment:
Out of curiosity, do these eng/script powershell files need a license header comment?